testutils/sims/runner.go

```diff
@@ -185,7 +189,7 @@ func NewSimulationAppInstance[T SimulationApp](
 	db, err := dbm.NewDB("Simulation", dbm.BackendType(tCfg.DBBackend), dbDir)
 	require.NoError(t, err)
 	t.Cleanup(func() {
-		require.NoError(t, db.Close())
+		_ = db.Close() // ensure db is closed
```

Comment:
The DB is closed by the app on on the happy path now so that errors can be ignored